ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml

```diff
@@ -80,11 +79,24 @@ presubmits:
         - --lease-server-password-file=/etc/boskos/password
         - --lease-server-username=ci
         - --lease-server=https://boskos-ci.svc.ci.openshift.org
+        - --repo=windows-machine-config-operator
         - --secret-dir=/usr/local/e2e-operator-cluster-profile
         - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
         - --target=e2e-operator
+        - --template=/usr/local/e2e-operator
         command:
         - ci-operator
+        env:
+        - name: CLUSTER_TYPE
```

Comment:
Why is this not present in the 4.5 job?

Reply:
Because we never added it there in the first place